src/parser/view_transform.c

```diff
@@ -3290,25 +3266,18 @@ mq_copypush_sargable_terms_helper (PARSER_CONTEXT * parser, PT_NODE * statement,
 int
 mq_copypush_sargable_terms (PARSER_CONTEXT * parser, PT_NODE * statement, PT_NODE * spec)
 {
-  PT_NODE *derived_table;
   int push_cnt = 0;		/* init */
   FIND_ID_INFO info;
+  PT_NODE *subquery = NULL;
 
-  if (statement->node_type == PT_SELECT
-      /* never do copy-push optimization for a hierarchical query */
-      && statement->info.query.q.select.connect_by == NULL && spec->info.spec.derived_table_type == PT_IS_SUBQUERY
-      && (derived_table = spec->info.spec.derived_table) && PT_IS_QUERY (derived_table)
-      && !PT_SELECT_INFO_IS_FLAGED (statement, PT_SELECT_INFO_IS_MERGE_QUERY))
-    {
-      info.type = FIND_ID_INLINE_VIEW;	/* inline view */
-      /* init input section */
-      info.in.spec = spec;
-      info.in.others_spec_list = statement->info.query.q.select.from;
-      info.in.attr_list = spec->info.spec.as_attr_list;
-      info.in.query_list = derived_table;
```

Comment:
info.in.query_list was set, but it is not set at the new change.
Is it not used anymore?

Reply:
The query_list was changed to subquery.